### DISCUSSION TITLE
Unable to login Facebook

### DISCUSSION THREAD

**Discussion by sffahad08** [Authenticator-Extension/Authenticator#763] [General]
Please help me

**Comment by Sneezry** [comment]
If Facebook keeps showing error code, try sync clock with Google in settings menu.

If you lost your code, see https://authenticator.cc/docs/en/lost-codes

**Comment by sffahad08** [comment]
Thanks for your response but it doesn't seems easy. Whatever I do all it
comes on the same page and I can't bypass 2 step verification.

**Reply by Sneezry** [reply]
It seems more like a Facebook issue than Authenticator.
